src/generators/typescript/TypeScriptGenerator.ts

```diff
@@ -41,13 +42,19 @@ export class TypeScriptGenerator extends AbstractGenerator<TypeScriptOptions,Typ
     defaultPreset: TS_DEFAULT_PRESET,
     typeMapping: TypeScriptDefaultTypeMapping,
     constraints: TypeScriptDefaultConstraints,
-    moduleSystem: 'ESM'
+    moduleSystem: 'ESM',
+    dependencyManagerFactory: undefined
+  };
+  
+  static defaultCompleteModelOptions: TypeScriptRenderCompleteModelOptions = {
+    exportType: 'default'
   };
 
   constructor(
     options?: DeepPartial<TypeScriptOptions>,
   ) {
     const realizedOptions = mergePartialAndDefault(TypeScriptGenerator.defaultOptions, options) as TypeScriptOptions;
+    realizedOptions.dependencyManagerFactory = () => { return new TypeScriptDependencyManager(realizedOptions);};
```

Comment:
Here you override `dependencyManagerFactory` field and people should have option to pass their manager.

Reply:
Right... Good catch!